code/modules/uplink/uplink_items.dm

```diff
@@ -389,6 +389,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 40
 	include_modes = list(/datum/game_mode/nuclear)
 
+/datum/uplink_item/implants/knuckleblaster
+	name = "Knuckleblaster Arm"
+	desc = "A robotic arm capable of shooting shotgun shells as well as devastating melee attacks. Comes with a free autosurgeon."
+	item = /obj/item/autosurgeon/organ/knuckleblaster
+	cost = 10
```

Comment:
10 tc seems cheap for a free concealable shotgun

Reply:
I have to disagree, that's half of a traitor's money for something that's single shot and concealable, so basically a way sneakier and slightly more reliable version of the Improvised shotgun, and considering how easy those are to make, I'd have to argue that increasing the price too much will make it too expensive when you can get a similar effect by making a sawn off improvised, which is only slightly worse, I may be wrong but I don't think this thing's going to be as effective as a Esword, but if it did need a nerf, I'd recommend giving it a spread at range, like the sawn off mosin to make it more of a ambush weapon that loses effectiveness at range due to the smaller barrel.

Reply:
Ok so the reason why it should be raised above 10tc is that buckshot is crazy easy to produce at a hacked lathe, and this is a retractable (able to be hidden) implant which is very strong. and buckshot does something like 80-90 damage if all the pellets hit and it does 15 damage without a shotgun shell loaded which is very strong.